spatialscaper/utils.py

```diff
@@ -582,24 +582,21 @@ def traj_2_ir_idx(XYZs, trajectory):
     return indices
 
 
-def db2scale(db):
+def db2multiplier(db, x):
```

Comment:
Shall we leave `x=1` as default? I am thinking it could simplify things for some users.